### DISCUSSION TITLE
Cant fetch database info.

### DISCUSSION THREAD

**Discussion by TiagoRato13** [vercel/community#3143] [Help]
### Summary

I have my website backend deployed in cyclic and frontend in vercel.
When I open the vercel link to my app it opens but gives me the error "Failed to load resource: the server responded with a status of 404 ()" is this related to the frontend or backend? how can I fix the issue? thanks in advance

### Example

https://tiagorato.vercel.app/

### Steps to Reproduce

Inspect is enough to check the error

**Comment by anttituomola** [comment]
Your error message seems to be {"message":"This route does not exist"}. Can you check if you have typo in the API call or in the server route?

**Reply by TiagoRato13** [reply]
I've been trying several ways and names but none seems to be giving me no error. I'm really lost here

**Comment by tobyab** [comment]
Would you mind sharing a link to the repo(s)?
